src/core/qgsquick/qgsquickcoordinatetransformer.cpp

```diff
@@ -127,6 +144,62 @@ void QgsQuickCoordinateTransformer::updatePosition()
   if ( mSkipAltitudeTransformation )
     z = mSourcePosition.z();
 
+  QSettings settings;
+  const QString verticalGridName = settings.value( QStringLiteral( "verticalGrid" ), QString() ).toString();
+  if ( !verticalGridName.isEmpty () )
+  {
+    if ( mVerticalGridName != verticalGridName )
+    {
+#if defined(Q_OS_ANDROID)
```

Comment:
Hmmm... that needs a bit of reworking.

Currently we have platformUtils defined in qgismobileapp.cpp. This needs to be changed.

```

#if defined (Q_OS_ANDROID)
Q_GLOBAL_STATIC(AndroidPlatformUtilities, sPlatformUtils)
#else ...
...
#endif

PlatformUtilities *PlatformUtilities::instance()
{
  return sPlatformUtils;
}
```

And then always use `PlatformUtilities::instance()` everywhere. This way we can register new platforms at a single point in the source code.